synapse/push/pusher.py

```diff
@@ -56,7 +56,7 @@ def create_pusher(self, pusherdict):
         f = self.pusher_types.get(kind, None)
         if not f:
             return None
-        logger.info("creating %s pusher for %r", kind, pusherdict)
+        logger.debug("creating %s pusher for %r", kind, pusherdict)
```

Comment:
I'm not entirely convinced by this. it's the only thing that gets logged during minutes of cpu-spinning, and it's not *massively* high-volume.

Reply:
It's logging several times a millisecond with a reasonably a non-trivial dict, which feels a bit OTT. There are ~170K pushers, with a log length of ~600 characters, means about 102MB of logs

I could make it so it logs in batches the progress?

Reply:
fair enough. let's see how it looks.